components/lib/badge/__snapshots__/Badge.spec.js.snap

```diff
@@ -1,78 +1,78 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Badge check snapshots: default 1`] = `
+exports[`Badge default 1`] = `
```

Comment:
Here the 'check snapshots: ' string was not really useful, as it is clear that snapshots are being checked.